tests/e2e/e2e_common.go

```diff
@@ -31,6 +31,7 @@ const (
 	adminUser                                  = "Administrator@vsphere.local"
 	apiServerIPs                               = "API_SERVER_IPS"
 	attacherContainerName                      = "csi-attacher"
+	windowsLTSC2019Image                       = "harbor-repo.vmware.com/csi/windows-servercore:ltsc2019"
```

Comment:
According to latest guidelines we should not add harbor repo details (Vmware Internal) to the codebase. This should be handled via Jenkins

Reply:
Declare a variable for this pod image in e2e_common.go, refer busyBoxImageOnGcr
Refer func init() in suite_test.go  to assign POD image